cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -381,7 +381,7 @@ private int runTestSuit(Path moduleTestCache, Target target, Collection<Path> te
         cmdArgs.add(target.path().toString());
         cmdArgs.add(orgName);
         cmdArgs.add(packageName);
-        cmdArgs.add(moduleName.isEmpty() ? "''" : moduleName); // see JDK-7028124
+        cmdArgs.add(moduleName.isEmpty() ? "\"\"" : moduleName); // see JDK-7028124
```

Comment:
Lets wrapt it always. It's more consistent.
```suggestion
        cmdArgs.add("\"" + moduleName + "\""); // see JDK-7028124
```